osu.Game/Rulesets/UI/RulesetInputManager.cs

```diff
@@ -34,11 +34,15 @@ protected override InputState CreateInitialState()
 
         protected readonly KeyBindingContainer<T> KeyBindingContainer;
 
-        protected override Container<Drawable> Content => KeyBindingContainer;
+        protected override Container<Drawable> Content => content;
+
+        private readonly Container content;
 
         protected RulesetInputManager(RulesetInfo ruleset, int variant, SimultaneousBindingMode unique)
         {
-            InternalChild = KeyBindingContainer = CreateKeyBindingContainer(ruleset, variant, unique);
+            InternalChild = KeyBindingContainer =
+                (KeyBindingContainer<T>)CreateKeyBindingContainer(ruleset, variant, unique)
```

Comment:
Won't be needed after a framework update

```suggestion
                CreateKeyBindingContainer(ruleset, variant, unique)
```